src/Illuminate/Cookie/CookieJar.php

```diff
@@ -30,6 +30,13 @@ class CookieJar implements JarContract
     protected $secure = false;
 
     /**
+     * The default httpOnly setting (defaults to true).
+     *
+     * @var bool
+     */
+    protected $httpOnly = true;
```

Comment:
As I mentioned in <https://github.com/laravel/framework/pull/12818#discussion_r57085319>

Making this default to `true` while previously it's default to `false` in a patch release should be consider as breaking change.